src/uct/ib/ud/base/ud_iface.c

```diff
@@ -437,6 +436,14 @@ UCS_CLASS_INIT_FUNC(uct_ud_iface_t, uct_ud_iface_ops_t *ops, uct_md_h md,
     self->config.check_grh_dgid  = config->dgid_check &&
                                    uct_ib_iface_is_roce(&self->super);
 
+    if (config->slow_timer_tick <= 0.) {
+        ucs_error("The slow timer tick should be > 0 (%lf)",
+                  config->slow_timer_tick);
+        return UCS_ERR_INVALID_PARAM;
+    } else {
+        self->config.slow_timer_tick = config->slow_timer_tick;
```

Comment:
could init  `iface->async.slow_tick` here, then no need to keep `self->config.slow_timer_tick`